src/GSoft.Extensions.MediatR/MediatorBuilder.cs

```diff
@@ -1,54 +1,87 @@
 using System.Reflection;
 using MediatR;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.DependencyInjection.Extensions;
 
 namespace GSoft.Extensions.MediatR;
 
 public sealed class MediatorBuilder
```

Comment:
Lots of changes here due to MediatR 12.x breaking changes about how to add the library.
Hopefully these breaking change will be abstracted by us in this library, so consumers won't be impacted in the future.